### DISCUSSION TITLE
'Inherited Owner' role definition

### DISCUSSION THREAD

**Discussion by eudesbarbosa** [bihealth/sodar-server#1484] [Ideas]
## Inherited Owner

The role and level of permission of an 'Inherited Owner' definition could be improved in the 'Members' tab and in the manual...

I accidentality asked a project owner to add me as 'Contributor' to a project that I'm the 'Inherited Owner'. 
The following happened:
1. From the info in 'Members' it wasn't clear to the user what I could do the project nor could the user change my role in the project. 
2. The user tried to  'Add member...'  and could not find my name in the search box.

Is there a way to clarify that?

**Comment by mikkonie** [comment]
Project management functionality is implemented in [SODAR Core](https://github.com/bihealth/sodar-core), so this should be discussed in that repository. Also, the inheritance functionality will change in SODAR Core v0.12.

I can see about clarifying role inheritance in the SODAR documentation and the SODAR Core tour help. In short, there is no reason to grant an inherited owner a contributor role, as it would simply reduce their existing permissions if allowed.

**Comment by eudesbarbosa** [comment]
Just clarifying the role already does the trick, no need to change the underlying definition.

Another thing that I was checking today, other roles explanations in the manual could be expanded. 
What I would consider:
1. Include can authorize data deletion;
2. Expand what is expected from the Owner and Delegate, you could use Academic examples. Something like: _"Owner, usually the principal investigator, very little time to deal with day-to-day operations but ultimately responsible for the data. Delegate, could be another principal investigator (collaborations) or a research assistant responsible for the daily operations and actual execution of the project."_

https://sodar-server.readthedocs.io/en/latest/ui_project_members.html#member-roles

PS: I don't think I ever started an issue or discussion in the correct SODAR repo XD
